src/frontend/styles/layout.pcss

```diff
@@ -21,8 +20,17 @@
     }
   }
 
+  &__aside-right {
+    position: fixed;
+    top: var(--layout-height-header);
+    right: 0;
+    width: var(--layout-width-aside-right);
+    max-width: var(--layout-width-aside-right);
```

Comment:
on narrow screens right aside is still rendered, but as grey line 1px wide. Let's make it take full width and appear under main content on mobile?
<img width="461" alt="image" src="https://user-images.githubusercontent.com/31101125/174970875-fc3d3bc2-1ac9-42bc-b75d-e819b307f9e3.png">